content/writing/checking-in.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: "Starting Meetings Well With A Check In"
+date: 2020-10-26
+intro: Starting meetings with a check in can be really helpful. Here's how.
+---
+
+Before every meeting we do, we like to check in. This means going from person to person (we call this talking in rounds), saying out loud how we are feeling, if there is anything on our mind and how things are going. We have done this in more or less every meeting we have done since our founding in 2018.
+
+For example, a standard check in might run something like:
+
+- Person A: Feeling great, looking forward to the meeting
+- Person B: A bit anxious at the moment, worried about the contracts coming in
+- Person C: Feel pretty great, excited for the meeting
+- Person D: Feeling quite tired, been a long week, bit worried because my partner is sick.
+
+## Why we do this
+
+This ritual has two effects:
+
+1. It lets people other people in the meeting know what is going on outside of the meeting.
+2. It allows space for these things to be said, when they might have otherwise not been.
+
+Check ins allow us to adjust our behaviour according to what's been shared by other in the group, and act in solidarity towards them. For example, there might be something that we can do to alleviate someone's anxiety. The fact that one member is tired might mean you are especially tight on meeting timings throughout so you can end the meeting early.
+
+It also allows people to get something that is worrying off their chest. A problem solved is a problem halved. It tries to make sharing difficult things less awkward by always allowing dedicated time for them.
+
+Someone saying they are worried about a difficult meeting immediately changes the dynamics of that meeting. If everyone admits it is likely going to be a difficult meeting, it makes it a lot easier to deal with.
+
+Speaking aloud outside worries allows you perhaps address them and, at least temporarily, put them aside. This will also mean that the meeting itself remains focused, as you've addressed any distractions at the outset.
```

Comment:
Minded to change this to "things that can be distracting" as distractions has a bit of a normative charge.